### PR TITLE
Color system overhaul

### PR DESCRIPTION
**Pull request by dkieffer**
- new, expanded color palette
- more universal theme coverage within existing components
- new dark mode theme. auto activates based on user system prefs

### CODE REVIEW

File: frontend/src/components/elements/text-input.scss [hunk 1]
```diff
@@ -34,13 +34,15 @@
         height: 56px;
         box-sizing: border-box;
         padding: 14px 0px;
+        background-color: var(--background-color);
```
Comment (jonnytown) [inline frontend/src/components/elements/text-input.scss:37]: Is input supposed to looks like this? It's pretty intense in FF on MacOS.

image

File: frontend/src/components/elements/textarea.scss [hunk 2]
```diff
@@ -33,18 +33,20 @@
         width: 100%;
         height: 56px;
         min-height: 56px;
+        background-color: var(--background-color);
```
Comment (jonnytown) [inline frontend/src/components/elements/textarea.scss:36]: Maybe this is where my other comment should be?

Comment (dkieffer) [reply frontend/src/components/elements/textarea.scss:36]: I can't reproduce. We may need to screenshare

Comment (jonnytown) [reply frontend/src/components/elements/textarea.scss:36]: After rebuilding the docker images everything looks great!

File: frontend/src/components/elements/text-input.scss (hunk 1 above)
Comment (jonnytown) [reply frontend/src/components/elements/text-input.scss:37]: just docker issues on my side

**Review by jonnytown — APPROVED** [review]

**Pull request merged**